athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandler.java

```diff
@@ -59,18 +61,37 @@ public class NeptuneRecordHandler extends RecordHandler
      * catalog id to correlate relevant query errors.
      */
     private static final String SOURCE_TYPE = "neptune";
-    private final NeptuneConnection neptuneConnection;
+    private NeptuneConnection neptuneConnection = null;
+    private java.util.Map<String, String> configOptions = null;
+
+    private static NeptuneConnection createConnection(java.util.Map<String, String> configOptions) 
+    {
+        GraphType graphType = GraphType.PROPERTYGRAPH;
+        if (System.getenv("neptune_graphtype") != null) {
```

Comment:
use configOptions.getEnv here. We dump System.getEnv to configOptions when the RecordHandler is instantiated.

Reply:
changed